src/Type.cpp

```diff
@@ -71,7 +72,20 @@ Type *Type::void_type = NULL;
 static vector<Type *> AllTypes;
 static vector<Type *> derived_types;
 
+AttributeGenerator type_attr_generator;
+
 //////////////////////////////////////////////////////////////////////
+
+void
+TypeAttributes()
+{
+	type_attr_generator.attributes.push_back(new MultiValuedAttribute("visibility", TypeAttrProb, {"default", "hidden", "protected", "internal"}));
+	type_attr_generator.attributes.push_back(new AlignedAttribute("aligned", TypeAttrProb, 8));
+	type_attr_generator.attributes.push_back(new AlignedAttribute("warn_if_not_aligned", TypeAttrProb, 8));
```

Comment:
This must be a typo.

Reply:
I didn't get it what do you mean. Are you telling typo about `warn_if_not_aligned`?

I am using Aligned class for it so that it will generate a alignment upto 2^8

Reply:
I see, `warn_if_not_aligned` would be then fine :)